addOns/graphql/src/main/java/org/zaproxy/addon/graphql/GraphQlFingerprinter.java

```diff
@@ -47,7 +47,7 @@ public class GraphQlFingerprinter {
     private static final Logger LOGGER = LogManager.getLogger(GraphQlFingerprinter.class);
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
-    private static List<DiscoveredGraphQlEngineHandler> handlers;
+    private static List<DiscoveredGraphQlEngineHandler> handlers = new ArrayList<>(2);
```

Comment:
This could probably even be 1, it's unlikely to be incremented past one. Even if it is, it would be infrequent.